### PR TITLE
100% `net/minecraft/world/gen`

### PR DESCRIPTION
**Pull request by OroArmor**
I am mentally stable

### CODE REVIEW

File: mappings/net/minecraft/world/gen/DensityFunctions.mapping [hunk 1]
```diff
@@ -32,6 +42,8 @@ CLASS net/minecraft/unmapped/C_mdkmcezi net/minecraft/world/gen/DensityFunctions
 		ARG 1 min
 		ARG 3 max
 	METHOD m_irznzryl singleArgumentCodec (Lcom/mojang/serialization/Codec;Ljava/util/function/Function;Ljava/util/function/Function;)Lnet/minecraft/unmapped/C_ircwepir;
+		ARG 1 into
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/world/gen/DensityFunctions.mapping]:
```suggestion
		ARG 1 to
```
?

File: mappings/net/minecraft/world/gen/chunk/Blender.mapping [hunk 2]
```diff
@@ -26,10 +29,22 @@ CLASS net/minecraft/unmapped/C_pfqozslu net/minecraft/world/gen/chunk/Blender
...
+		ARG 6 x
+		ARG 7 y
+		ARG 8 z
+		ARG 9 density
 	METHOD m_iflcqbrq (Lnet/minecraft/unmapped/C_nrynfjsf$C_uavkwagl;Lnet/minecraft/unmapped/C_nrynfjsf;)V
 		ARG 1 mask
 	METHOD m_ikgldbtv heightToOffset (D)D
 		ARG 0 height
+	METHOD m_incvmsck (IILorg/apache/commons/lang3/mutable/MutableDouble;Lorg/apache/commons/lang3/mutable/MutableDouble;Lorg/apache/commons/lang3/mutable/MutableDouble;Ljava/lang/Long;Lnet/minecraft/unmapped/C_jfjllmkc;)V
+		ARG 5 l
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/world/gen/chunk/Blender.mapping]:
```suggestion
		ARG 5 chunkPos
```

File: mappings/net/minecraft/world/gen/chunk/BlendingData.mapping [hunk 3]
```diff
@@ -18,39 +20,87 @@ CLASS net/minecraft/unmapped/C_jfjllmkc net/minecraft/world/gen/chunk/BlendingDa
...
 	METHOD m_nxzfejge getHeight (III)D
 		ARG 1 x
 		ARG 2 y
 		ARG 3 z
+	METHOD m_nyjnuisq acceptHeights (IILnet/minecraft/unmapped/C_jfjllmkc$C_bnlexvzr;)V
+		ARG 1 minHalfSectionY
+		ARG 2 maxHalfSectionY
+		ARG 3 consumer
+	METHOD m_opdlcxgw getOldHeightLimit ()Lnet/minecraft/unmapped/C_qpninoyb;
+	METHOD m_pslasnyw zeroIfNegative (I)I
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/world/gen/chunk/BlendingData.mapping]:
```suggestion
	METHOD m_pslasnyw getMaxWith0 (I)I
```
idak if I like this...
but it should at least have `get`

**Review by supersaiyansubtlety — CHANGES_REQUESTED** [review]

File: build.gradle [hunk 4]
```diff
@@ -393,7 +393,7 @@ javadoc {
 				"https://javadoc.io/doc/io.netty/netty-all/${project.netty}/",
 				'https://javadoc.lwjgl.org/',
 				'https://logging.apache.org/log4j/2.x/javadoc/log4j-api/',
-				'https://fastutil.di.unimi.it/docs/',
+				// 'https://fastutil.di.unimi.it/docs/',
```
Comment (OroArmor) [inline build.gradle]:
```suggestion
				'https://fastutil.di.unimi.it/docs/',
```

**Review by supersaiyansubtlety — APPROVED** [review]

**Review by eristhea — APPROVED** [review]
looks good! i love how often you just remove mappings for automapped stuff

**Pull request merged**